tests/python/frontend/onnx/test_forward.py

```diff
@@ -142,6 +142,37 @@ def test_reshape():
     tvm.testing.assert_allclose(ref_shape, tvm_out.shape)
 
 
+def test_expand():
+    in_shape = (1, 3, 1, 4)
+    shape = (4, 3, 3, 4)
```

Comment:
It would be better if we can add `dim_changed` and `dim_unchanged` test cases.

Reply:
does that mean to add both cases that the output shape will be changed by following the given "shape" and not ??

Reply:
Yes, because it is possible that the output.shape is not equal to the shape.